tensorflow/lite/micro/kernels/cmsis_nn/conv.cc

```diff
@@ -276,21 +427,6 @@ TfLiteStatus Eval(TfLiteContext* context, TfLiteNode* node) {
       return EvalQuantizedPerChannel(context, node, params, data, input, filter,
                                      bias, output, nullptr);
       break;
-    case kTfLiteInt16: {
```

Comment:
let's not remove int16 from the most generic Eval. We still want Register_CONV_2D() to support all the data types. Instead let's call Eval16x8 from here.

Reply:
Ok. Perhaps we should change to CONV_2D_INT8 for the rest of the examples as well (maybe in another PR) to keep the memory footprints down?